resteasy-jaxrs/src/main/java/org/jboss/resteasy/core/ServerResponseWriter.java

```diff
@@ -408,9 +409,16 @@ public static void commitHeaders(BuiltResponse jaxrsResponse, HttpResponse respo
                   response.addNewCookie(cookie);
                   it.remove();
                }
+               else if (next instanceof Object)

```

Comment:
`next instance Object` will always be true right ?

Reply:
Sadly yes.  I am testing a change for this.